src/entities/Resource.js

```diff
@@ -40,6 +41,9 @@ export default class Resource {
    */
   setDesc(value) {
     this.desc = value;
+    if (value !== undefined) {
```

Comment:
I think better to use comparison like `if (!value)`

Reply:
If we use `!value`, it will be true/false but `value` is still 'undefined'. So in the next line, `marked(value)` will throws an error.